### ISSUE TITLE
Remove SYM-CAK, Add SM-AUTH

### ISSUE THREAD

**Issue by currydm** [usnistgov/FIPS201#297]
### All Fields Are Required
**Organization Name (N/A, if individual)**: SSA

**Organization Type (see below for codes)**: 1 - Federal
 
**Reference (Include section/paragraph or pdf line number)**: 6.3.1 Table 6-1
 
**Comment (Include rationale for comment)**: "1. ) SYM-CAK has been deprecated and is included in the table
2.) SM-AUTH should be added to the table"

 
**Suggested Change**: Remove SYM-CAK, Add SM-AUTH

 
---
 
Organization Type: 1 = Federal, 2 = Industry, 3 = Academia, 4 = Self, 5 = Other

**Comment by jricher** [comment]
Suggest accept, VIS (also deprecated) was removed from the table's equivalent in the old draft. Where does SM-AUTH fit?

**Comment by hferraio** [comment]
Our definition of "deprecated in line 417-419 states:  

_"Deprecated features MAY continue to be used but SHOULD be phased out in future systems since the feature will likely
be removed in the next revision of the Standard"_  

while Removed features: 

_Removed features SHALL NOT be used. For example, the CHUID authentication mechanism (Section 6.2.5) has been removed from this version of the Standard and relying systems SHALL NOT use this authentication _mechanism____

Since deprecated features are in 'transition', they are still used until phased out. So, to delineate  from REMOVED, maybe deprecated should still be listed.  This is the way we've handled it with CHUID AuthN.
